L1Trigger/TrackFindingTracklet/src/TrackletLUT.cc

```diff
@@ -901,6 +901,7 @@ void TrackletLUT::initProjectionDiskRadius(int nrbits) {
 void TrackletLUT::initBendMatch(unsigned int layerdisk) {
   unsigned int nrinv = NRINVBITS;
   double rinvhalf = 0.5 * ((1 << nrinv) - 1);
+  assert(layerdisk < 16);  // should be a named constant for this
```

Comment:
Not urgent from my side, but if we know this is a bit "magic number"-ish, should we just go ahead and make this a constant?

Reply:
I am far from being an expert of this code, but that `16` to me looks like the total number of layer/disks in the phase2 layout. I.e. `2 * N_DISK + N_LAYER`: 

https://github.com/cms-sw/cmssw/blob/c1580c857f90b5607eefc4608b31df92b7cdfd78/L1Trigger/TrackFindingTracklet/interface/Settings.h#L25-L26